docs/en/observability/synthetics-quickstart-fleet.asciidoc

```diff
@@ -48,10 +48,14 @@ beta[] Elastic Synthetics tests can be configured through {fleet} using the Elas
 image::images/synthetics-integration.png[Synthetics integration]
 
 In the example above, our library of synthetic tests is downloaded from the
-remote zip endpoint for https://github.com/elastic/synthetics-demo/tree/main/todos/synthetics-tests[our todos example]. 
+remote zip endpoint for the git repository for https://github.com/elastic/synthetics-demo/tree/main/todos/synthetics-tests[our todos example]. 
+
+By using a git repository for our synthetics journeys we can have a gitops workflow for deploying changes to our synthetic monitors, letting us features like
+pull requests, reverts, etc. for managing our testing.
+
 The fields should be set as follows:
 
-* *Zip URL*: URL pointing to a zip file containing the test project. 
+* *Zip URL*: URL pointing to a zip file containing the test project.
 
 * *Folder*: The folder within the zip file containing synthetics tests, this will be the working directory in which the synthetics agent will be run. {heartbeat} will invoke the synthetics library in this folder. 
```

Comment:
```suggestion
* *Folder*: The folder within the zip file containing synthetics tests. This will be the working directory in which the synthetics agent runs. {heartbeat} will invoke the synthetics library in this folder. 
```